quick-and-easy-faqs.php

```diff
@@ -28,6 +28,11 @@
 
 define( 'QUICK_AND_EASY_FAQS_VERSION', get_plugin_data( __FILE__ )['Version'] );
 
+// Plugin file path relative to plugins directory.
+if ( ! defined( 'QUICK_AND_EASY_FAQS_BASENAME' ) ) {
```

Comment:
Why do you think it will be defined already?

Reply:
Just to eliminate even slight change of a plugin conflict in future.